docs/get-started/deployment-and-usage.md

```diff
@@ -0,0 +1,297 @@
+as a template:
+
+```bash
+cp nfd-worker.conf.example nfd-worker.conf
+vim nfd-worker.conf  # edit the configuration
+kubectl create configmap nfd-worker-config --from-file=nfd-worker.conf
+```
+
+Then, configure Volumes and VolumeMounts in the Pod spec (just the relevant
+snippets shown below):
+
+```yaml
+...
+  containers:
+      volumeMounts:
+        - name: nfd-worker-config
+          mountPath: "/etc/kubernetes/node-feature-discovery/"
+...
+  volumes:
+    - name: nfd-worker-config
+      configMap:
+        name: nfd-worker-config
+...
+```
+
+You could also use other types of volumes, of course. That is, hostPath if
+different config for different nodes would be required, for example.
+
+The (empty-by-default)
+[example config](https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/nfd-worker.conf.example)
+is used as a config in the NFD Docker image. Thus, this can be used as a default
+configuration in custom-built images.
+
+Configuration options can also be specified via the `--options` command line
+flag, in which case no mounts need to be used. The same format as in the config
+file must be used, i.e. JSON (or YAML). For example:
+
+```
+--options='{"sources": { "pci": { "deviceClassWhitelist": ["12"] } } }'
+```
+
+Configuration options specified from the command line will override those read
+from the config file.
+
+Currently, the only available configuration options are related to the
+[CPU](#cpu-features), [PCI](#pci-features) and [Kernel](#kernel-features)
+feature sources.
+
+## Using Node Labels
+
+Nodes with specific features can be targeted using the `nodeSelector` field. The
+following example shows how to target nodes with Intel TurboBoost enabled.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    env: test
+  name: golang-test
+spec:
+  containers:
+    - image: golang
+      name: go1
+  nodeSelector:
+    feature.node.kubernetes.io/cpu-pstate.turbo: 'true'
+```
+
+For more details on targeting nodes, see
+[node selection](https://kubernetes.io/docs/tasks/tools/install-kubectl).
+
+## Uninstallation
+
+### Operator Was Used for Deployment
+
+If you followed the deployment instructions above you can simply do:
+
+```bash
+kubectl -n nfd delete NodeFeatureDiscovery my-nfd-deployment
+```
+
+Optionally, you can also remove the namespace:
+
+```bash
+kubectl delete ns nfd
+```
+
+See the [node-feature-discovery-operator][nfd-operator] and [OLM][OLM] project
+documentation for instructions for uninstalling the operator and operator
+lifecycle manager, respectively.
+
+### Manual
+
+```bash
+NFD_NS=node-feature-discovery
+kubectl -n $NFD_NS delete ds nfd-worker
+kubectl -n $NFD_NS delete deploy nfd-master
+kubectl -n $NFD_NS delete svc nfd-master
+kubectl -n $NFD_NS delete sa nfd-master
+kubectl delete clusterrole nfd-master
+kubectl delete clusterrolebinding nfd-master
+```
+
+### Removing Feature Labels
+
+Nfd-master has a special `--prune` command line flag for removing all
```

Comment:
Nfd looks weird, you have written above NFD-Master?

Reply:
I now tried to unify this, using **NFD-Master** and **NFD-Worker** in headings and in the beginning of sentences.